staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -165,6 +166,7 @@ func NewAdmissionController(
 }
 
 func (c *celAdmissionController) Run(stopCh <-chan struct{}) {
+	// TODO: Doesn't this comparison need a lock?
```

Comment:
yeah, this appears to be supporting stopping and then restarting the admission controller, or tolerating multiple calls to Run(), neither of which are things I would expect.

@alexzielenski, can you add this to the list of bits to revisit and maybe remove?